MVP Module/Source/Feed UI/Composers/FeedUIComposer.swift

```diff
@@ -21,6 +21,7 @@ public final class FeedUIComposer {
 				controller: feedController,
 				imageLoader: MainQueueDispatchDecorator(decoratee: imageLoader)),
 			loadingView: WeakRefVirtualProxy(feedController))
+		presentationAdapter.errorPresenter = FeedErrorPresenter(feedErrorView: FeedErrorViewAdapter(controller: feedController), loadingView: WeakRefVirtualProxy(feedController))
```

Comment:
There's no need to create a separate `errorPresenter` and another `Adapter`. It's simpler to make the `FeedViewController` implement the `errorView` protocol, since the viewController is already responsible for displaying the view.
```suggestion
				controller: feedController,
				imageLoader: MainQueueDispatchDecorator(decoratee: imageLoader)),
			loadingView: WeakRefVirtualProxy(feedController),
			errorView: WeakRefVirtualProxy(feedController))
```

Reply:
resolved